.vib/elasticsearch/cypress/cypress/integration/elasticsearch_spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+import body from '../fixtures/documents.json';
+
+it('can get cluster info', () => {
+  cy.request({
+    method: 'GET',
+    url: '/',
+    form: true,
+  }).then((response) => {
+    expect(response.status).to.eq(200);
+    expect(response.body.cluster_name).to.contain('elastic');
+    expect(response.body.tagline).to.contain('You Know, for Search');
+  });
+});
+
+it('can check cluster health', () => {
+  const NODE_NUMBER = Cypress.env('nodeNumber');
+  cy.request({
+    method: 'GET',
+    url: '/_cluster/health',
+    form: true,
+  }).then((response) => {
+    expect(response.status).to.eq(200);
+    expect(response.body.status).to.contain('green');
+    expect(response.body.number_of_nodes).to.eq(NODE_NUMBER);
+  });
+});
+
+it('can get nodes stats', () => {
+  cy.request({
+    method: 'GET',
+    url: '/_nodes/stats',
+    form: true,
+  }).then((response) => {
+    expect(response.status).to.eq(200);
+    expect(response.body._nodes.successful).to.eq(4);
```

Comment:
You should be using the `nodeNumber` env here too

Reply:
Done!